mobile/src/navigation/types.ts

```diff
@@ -1,5 +0,0 @@
-export type RootStackParamList = {
```

Comment:
Removed this file because we no longer use this. It was initially used for navigation in the initial PRs.